package.json

```diff
@@ -30,43 +30,36 @@
     ],
     "repository": {
         "type": "git",
-        "url": "git+https://github.com/box/box-ui-elements.git"
+        "url": "git@github.com:box/box-ui-elements.git"
     },
     "bugs": {
         "url": "https://github.com/box/box-ui-elements/issues"
     },
     "scripts": {
-        "build": "yarn run pre-build && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev webpack --progress --colors --config build/webpack.config.js --mode development",
-        "build-ci": "LANGUAGE=en-US REACT=true yarn run build-prod",
-        "build:es": "./node_modules/.bin/babel src -d lib",
-        "build-prod": "NODE_ENV=production webpack --config build/webpack.config.js --mode production",
-        "build-npm": "yarn run npm && yarn run build-lib",
-        "build-link": "yarn run npm && yarn run build-link-lib",
-        "build:link-dev": "yarn run npm && yarn run build-link-lib -w",
-        "build-link-lib": "yarn run clean:lib && BABEL_ENV=dev NODE_ENV=dev yarn run build:es --ignore \"**/__tests__/**\",\"src/components/messages.js\" --copy-files -s inline",
-        "build-lib": "yarn run clean:lib && BABEL_ENV=npm NODE_ENV=production yarn run build:es --ignore \"**/__tests__/**\"",
+        "build": "yarn build:dev:dist",
+        "build:ci:es": "yarn build:prod:es",
+        "build:ci:dist": "LANGUAGE=en-US REACT=true yarn build:prod:dist",
+        "build:dev:dist": "LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev webpack --config build/webpack.config.js --mode development --progress --colors",
+        "build:es": "babel src --out-dir lib",
+        "build:es:link": "BABEL_ENV=npm NODE_ENV=dev yarn build:es --copy-files --source-maps inline --watch --ignore \"**/__tests__/**\",\"src/components/messages.js\"",
```

Comment:
It looks like this was changed from `BABEL_ENV=dev` to `BABEL_ENV=npm`. Not sure if it was intentional, but this will drop propTypes injection, I think.

Reply:
This script is kind of an odd one out now, as all the other namespaces are `action:environment:scope` whereas this is `action:scope:modifier`. Should this be something like `build:dev:es`?